internal/handlers/irc/irc.go

```diff
@@ -45,7 +47,9 @@ func (c Client) StartBot(errChan chan<- error, sendMessage func(string)) {
 	c.verbose.LogInfo("Starting up IRC bot...")
 	c.sendToTg = sendMessage
 	c.addHandlers()
-	if err := c.Connect(); err != nil {
+	// TODO: Currently just set to 5 seconds,
```

Comment:
I would like these comments to be removed before this PR merges, once we settle on a time.

Reply:
Yeah, that was the plan 👍

Reply:
I'd like to keep the comment that states it as a 5 second timeout but we can trim the second line

Reply:
@jwflory since `time` libraries are in every single language, I'm not sure we even need the 5 seconds clarification comment

Reply:
I am +0 on this one, not going to block on it :slightly_smiling_face: